x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/WaterfallContainer/Waterfall/waterfall_helpers/__test__/waterfall_helpers.test.ts

```diff
@@ -157,7 +157,8 @@ describe('waterfall_helpers', () => {
         transaction: {
           duration: { us: 49660 },
           name: 'GET /api',
-          id: 'myTransactionId1'
+          id: 'myTransactionId1',
+          start: { us: 1000 }
```

Comment:
I think `start` has been deprecated for `timestamp.us`. Fairly sure `start` is not sent up anymore.

Reply:
If the problem is that `timestamp.us` is not correct on legacy spans please add warning comments when where `start` is defined to ensure we don't use it by mistake in later versions.

Reply:
🤔Shouldn't it be used to create the timestamp when I ran the index migration process? Because the only way to align right the spans is with the start.

Reply:
Yes, I'd expect the migration script to have handled this.

Reply:
Try asking @graphaelli - I think he worked on the migration script.